File: polygot_playground/python/fundamental/variables.py
# Python variables
x, y, z = 10, 11, 12
xyz = 3298

def firstFunction():
    global xyz, abc, shouldAbc
    shouldAbc = True
    abc = xyz
    xyz = x + y + z
    isFnExecuted = True 
    return isFnExecuted

def secondFunction():
    if (shouldAbc):
        print(abc)
    if(isFnExecuted or isFirstFnExecuted): # error -> 'isFnExecuted' is not defined
        print('firstFunction Executed')

isFirstFnExecuted = firstFunction()
print(isFirstFnExecuted)
print(x + y + z)
print(x, y, z)
x = "10"
y = "11"
z = '12'
print(x + y + z)
print(x, y, z)
print(xyz)
if(isFirstFnExecuted):
    secondFunction()

"""
End
of
Program
"""

File: polygot_playground/javascript/fundamental/variables.mjs
/* 
    Javascript Variables 
    cmd: node variables.mjs
*/
let x = 10
let y = 11
let z = 21
let xyz = 3298
var shouldAbc = false

function firstFunction() {
    var shouldAbc = true
    let abc = xyz
    xyz = x + y + z
    return true
}

function secondFunction() {
    if(isFirstFnExecuted)
        console.log('firstFunction Executed')
    if (shouldAbc)
        console.log(abc)
}

const isFirstFnExecuted = firstFunction()
console.log(isFirstFnExecuted)
console.log(x + y + z)
console.log(x, y, z)
y = "11"
z = "12"
console.log(x + y + z)
console.log(x, y, z)
console.log(xyz)
if(isFirstFnExecuted)
    secondFunction()